pages/common/nimble.md

```diff
@@ -0,0 +1,28 @@
+# nimble
+
+> Package manager for the Nim programming language.
+> Manage Nim projects and their dependencies.
+
+- Search for packages:
+
+`nimble search {{search_string}}`
+
+- Install a package:
+
+`nimble install {{package_name}}`
+
+- List installed packages:
+
+`nimble list -i`
+
+- Create a new Nimble package in the current directory:
+
+`nimble init`
+
+- Build the Nimble package in the current directory:
```

Comment:
You don't have to repeat "in the current directory" after the first time. It is understood.